clients/rust/src/client.rs

```diff
@@ -257,82 +260,36 @@ impl Scope {
     pub fn new(usecase: Usecase) -> Self {
         Self(Ok(ScopeInner {
             usecase,
-            scope: String::new(),
+            scopes: scope_types::Scopes::empty(),
         }))
     }
 
-    fn for_organization(usecase: Usecase, organization: u64) -> Self {
-        let scope = format!("org={}", organization);
-        Self(Ok(ScopeInner { usecase, scope }))
+    fn for_organization(usecase: Usecase, organization: u64) -> crate::Result<Self> {
+        let scopes = scope_types::Scopes::from_iter([scope_types::Scope::create(
```

Comment:
`Scopes` was originally intended as immutable type. Now that we have a `push` method on it, we can use that over `from_iter`.